src/Model/MethodGo.cs

```diff
@@ -396,7 +396,7 @@ public IEnumerable<string> PrepareDecorators
                         }
                         else if (param.IsClientProperty)
                         {
-                            value = $"client.{param.Name.ToPascalCase()}";

```

Comment:
can we do a search all for `client.` to see if there are any other cases

Reply:
Yep that's how I found this one, I don't see any others.